examples/trading/api/helpers/create-demo-data.ts

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { TransactionBlock } from '@mysten/sui.js/transactions';
+
+import { CONFIG } from '../config';
+import { ACTIVE_NETWORK, getActiveAddress, signAndExecute } from '../sui-utils';
+
+// a simple example of objects by creating N amount of bears.
+const createDemoLockedObjects = async (totalBears: number) => {
+	if (totalBears < 5) throw new Error('Please create at least 5 bears to run this script.');
+	const txb = new TransactionBlock();
+	const bears = [];
```

Comment:
nit: `bears` is a little confusing as a name here, because it also ends up containing `Locked<DemoBear>`s and `Key`s as well -- can we tweak the name? How about: `toTransfer` or something?